core/rend/dx11/dx11_renderer.cpp

```diff
@@ -1294,6 +1306,94 @@ void DX11Renderer::writeFramebufferToVRAM()
 	WriteFramebuffer<2, 1, 0, 3>(width, height, (u8 *)tmp_buf.data(), texAddress, pvrrc.fb_W_CTRL, linestride, xClip, yClip);
 }
 
+#ifdef VIDEO_ROUTING
+void DX11Renderer::RenderVideoRouting()
+{
+	if (config::VideoRouting)
+	{
+		extern void os_VideoRoutingPublishFrameTexture(ID3D11Texture2D* pTexture);
+		
+		ID3D11RenderTargetView* pRenderTargetView = theDX11Context.getRenderTarget().get();
+
+		// Backbuffer texture would be different after reszing, fetching new address everytime
+		ID3D11Resource* pResource = nullptr;
+		pRenderTargetView->GetResource(&pResource);
+		ID3D11Texture2D* pTexture = nullptr;
+		pResource->QueryInterface(__uuidof(ID3D11Texture2D), (void**)&pTexture);
+		pResource->Release();
+		pTexture->Release();
```

Comment:
you release both `pResource` and `pTexture` interface pointers and use `pTexture` afterwards. Unless I'm missing something you should only call Release after you're done using the object